gradle/settings/parameters.gradle

```diff
@@ -0,0 +1 @@
+gradle.ext.serverProjectPath=":distributions"
```

Comment:
Really?  Distributions as a server base?  Seems odd.

Reply:
Switched back to ":"